src/@types/types.d.ts

```diff
@@ -17,7 +17,7 @@ interface Answer {
 interface Question {
     title: string;
     text: string;
-    options: Answer[];
+    options?: Answer[];
```

Comment:
is this question mark for open questions?

Reply:
Yes. I think that open questions will not have options field.